rules/windows/builtin/security/win_security_new_or_renamed_user_account_with_dollar_sign.yml

```
@@ -1,29 +1,30 @@
-title: New or Renamed User Account with '$' in Attribute 'SamAccountName'
+title: New or Renamed User Account with '$' Character
 id: cfeed607-6aa4-4bbd-9627-b637deb723c8
 status: test
-description: Detects possible bypass EDR and SIEM via abnormal user account name.
+description: |
+    Detects the creation of a user with the "$" character. This can be used by attackers to hide a user or trick detection system that lack the parsing mechanisms.
```

```suggestion
    Detects the creation of a user with the "$" character. This can be used by attackers to hide a user or trick detection systems that lack the parsing mechanisms.
```
what are "the parsing mechanisms"?